### ISSUE TITLE
Reloading Without Saving Changes Then Clicking "Stay on page" Disables Errors and Autosuggest

### ISSUE THREAD

**Issue by JavascriptFTW** [Khan/live-editor#319]
Recently while working on a program I accidentally tried reloading the page before saving, this, as it should, triggered the "Unsaved changes" dialog to which I answered "stay on page". But upon selecting "stay on page", saving, and continuing to edit my program, I discovered that Error Buddy, and auto suggest no longer worked. I tried this on some other programs, and the behavior persisted.
I used Firefox 38.0.5 on a machine running Windows 8.1
This bug may have been triggered due to my slow connection at the time since I don't remember experiencing it before.

**Comment by kevinb7** [comment]
Are there any errors in the console when this happens?

**Comment by JavascriptFTW** [comment]
No, there's nothing on the console other than the normal CSS and "Mutating the [[Prototype]] of an object" warnings which are always there.

**Comment by MikaalSky** [comment]
This appears to be killing absolutely everything related to auto-complete and such, which makes me think that whatever's controlling that is being disabled when the user hits refresh.
